admin/afs-client-accessd/afs-client-accessd

```diff
@@ -156,6 +156,15 @@ Example:
 
   AUDIT_PATH => '/usr/afs/logs/FileAudit',
 
+=item EXPORT_TIME
```

Comment:
I think this name might be a bit confusing, since it is not set at the "export time" but the hour of the day (in utc) the export is scheduled. Internally you've converted the value to be the epoch time by multiply the value the user gave by the number of seconds per day. 

I think it would be more clear to all this EXPORT_HOUR since it is the hour the export is scheduled.